notebooks/search/03-ELSER.ipynb

```diff
@@ -305,7 +307,7 @@
     "Let's note a few important parameters from that API call:\n",
     "\n",
     "- `inference`: A processor that performs inference using a machine learning model.\n",
-    "- `model_id`: Specifies the ID of the machine learning model to be used. In this example, the model ID is set to `.elser_model_1`.\n",
+    "- `model_id`: Specifies the ID of the machine learning model to be used. In this example, the model ID is set to `.elser_model_2`.\n",
     "- `target_field`: Defines the field where the inference result will be stored. Here, it is set to `ml`.\n",
```

Comment:
Hi @saarikabhasi! Should `target_field` here be updated to `output_field` to match [the JSON above](https://github.com/elastic/elasticsearch-labs/pull/72/files#diff-cefcd1253dbd13c7f622213b0e43e549c91cc4e4b4634996cb227e75f52ea5f8R291),

and maybe remove `text_expansion`, since `inference_config` isn't in the call anymore?

Reply:
Thanks @brianjolly. I might have missed to update the parameter description. Will update the PR to include the suggested change. Thanks again!